coalib/testing/LocalBearTestHelper.py

```diff
@@ -370,6 +371,8 @@ def verify_local_bear(bear,
     :param timeout:          Unused.  Use pytest-timeout or similar.
     :param tempfile_kwargs:  Kwargs passed to tempfile.mkstemp() if tempfile
                              needs to be created.
+    :param base_directory:   Path to the base directory. Changes the CWD to
+                             base_directory while running the bear.
```

Comment:
I am not quite sure if you need the second sentence it does not provide much information about the parameter in general.

Reply:
The second sentence gives a brief description as to what the parameter does. In my opinion it does clarify things.

Reply:
Ah okay but should it not be `This param is changed to CWD when the bear is running `?

Reply:
The wording is correct. CWD is changed according to the value of base_directory and not vice versa. Suppose I call a method with `base_directory=/opt/`, the function basically executes, `cd /opt/` and changes the CWD to `/opt/`